packages/web/src/brandkit/IconsExchangePage.tsx

```diff
@@ -0,0 +1,139 @@
+import { NextPage } from 'next'
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import CCLicense from 'src/brandkit/common/CCLicense'
+import { GAP } from 'src/brandkit/common/constants'
+import { brandStyles } from 'src/brandkit/common/constants'
+import Page, { ICONS_PATH } from 'src/brandkit/common/Page'
+import PageHeadline from 'src/brandkit/common/PageHeadline'
+import IconShowcase from 'src/brandkit/common/Showcase'
+import { AssetTypes, EXCHANGE_ICONS_PKG_TRACKING, trackDownload } from 'src/brandkit/tracking'
+import { NameSpaces, useTranslation } from 'src/i18n'
+import Button, { BTN } from 'src/shared/Button.3'
+import { hashNav } from 'src/shared/menu-items'
+import { colors } from 'src/styles'
+
+const icons = [
+  {
+    description: 'cUSD Exchange Icon, Full Color\n',
+    uri: '/images/marketplace-icons/icon-celo-dollar-color-f.svg',
```

Comment:
Can you change these so that they link to a zip file with both an svg and a png version of the file in it?